rsocket-examples/src/test/java/io/rsocket/integration/TcpIntegrationTest.java

```diff
@@ -28,21 +29,25 @@
     private AbstractRSocket handler;
 
     private Closeable server;
+    private InetSocketAddress address;
 
     @Before
     public void startup() {
+        TcpServerTransport serverTransport = TcpServerTransport.create("localhost", 0);
         server = RSocketFactory
             .receive()
             .acceptor((setup, sendingSocket) -> Mono.just(new RSocketProxy(handler)))
-            .transport(TcpServerTransport.create("localhost", 8000))
+            .transport(serverTransport)
```

Comment:
Ideally server would be a better type than Closeable, then I could get address off that.